en/orm/query-builder.rst

```diff
@@ -240,6 +240,15 @@ To set some basic conditions you can use the ``where()`` method::
     $query->where(['title' => 'First Post'])
         ->where(['published' => true]);
 
+You can also pass an anonymous function to the ``where()`` method. The passed anonymous function
+receives as it's first argument an instance of \Cake\Database\Expression\QueryExpression $exp
```

Comment:
you mean `its` here.

Reply:
See d1702f36c26f54cbbabd3860cb4ba4b438416513